src/runtime/audience-action-local-flow.ts

```diff
@@ -222,12 +239,14 @@ export class AudienceActionLocalFlow implements AudienceActionFlow {
     }
   }
 
-  private formSubmit_() {
+  private async formSubmit_() {
     //TODO: chuyangwang - verify email being submitted.
     this.eventManager_.logSwgEvent(
       AnalyticsEvent.ACTION_BYOP_NEWSLETTER_OPT_IN_SUBMIT
     );
-    this.complete_();
+    // Close the prompt.
+    this.unlock_();
```

Comment:
Can you also update the description to reflect these changes?

Reply:
Updated the description!